pages/_app.tsx

```diff
@@ -20,7 +21,7 @@ const App = ({ Component, pageProps }: AppProps) => (
       </SiteProvider>
     </LocaleProvider>
 
-    <Analytics />
+    {nextConstants.ENABLE_STATIC_EXPORT || <Analytics />}
```

Comment:
Doesn't look like this worked. There's still a request attempt to `/_vercel/insights/script.js`

Reply:
Indeed, probably because I evaluated this within the Component, which means it is evaluated during runtime. I'll make a hot-fix soon.

Reply:
Hmm, no easy way of evaluating this at build-time. Are these 404 too bad? I can honestly disable Vercel Analytics for now, anyways as we're not using it.

Reply:
I've pushed a hot-fix (one-liner) removing Analytics for now